setup.py

```diff
@@ -1,118 +1,144 @@
+        "csv = ibis.backends.csv",
+        "dask = ibis.backends.dask",
+        "hdf5 = ibis.backends.hdf5",
+        "impala = ibis.backends.impala",
+        "mysql = ibis.backends.mysql",
+        "pandas = ibis.backends.pandas",
+        "parquet = ibis.backends.parquet",
+        "postgres = ibis.backends.postgres",
+        "pyspark = ibis.backends.pyspark",
+        "spark = ibis.backends.pyspark",
+        "sqlite = ibis.backends.sqlite",
+    ]
+}
 
-all_requires = (
-    impala_requires
-    + postgres_requires
-    + mysql_requires
-    + kerberos_requires
-    + visualization_requires
-    + clickhouse_requires
-    + hdf5_requires
-    + parquet_requires
-    + spark_requires
-    + geospatial_requires
-    + dask_requires
-)
+setup_kwargs = {
+    "name": "ibis-framework",
+    "version": "1.4.0",
+    "description": "Productivity-centric Python Big Data Framework",
+    "long_description": "# Ibis: Python data analysis framework for Hadoop and SQL engines\n\n|Service|Status|\n| -------------: | :---- |\n| Documentation  | [![Documentation Status](https://img.shields.io/badge/docs-docs.ibis--project.org-blue.svg)](http://ibis-project.org) |\n| Conda packages | [![Anaconda-Server Badge](https://anaconda.org/conda-forge/ibis-framework/badges/version.svg)](https://anaconda.org/conda-forge/ibis-framework) |\n| PyPI           | [![PyPI](https://img.shields.io/pypi/v/ibis-framework.svg)](https://pypi.org/project/ibis-framework) |\n| GitHub Actions | [![Build status](https://github.com/ibis-project/ibis/actions/workflows/main.yml/badge.svg)](https://github.com/ibis-project/ibis/actions/workflows/main.yml) |\n| Coverage       | [![Codecov branch](https://img.shields.io/codecov/c/github/ibis-project/ibis/master.svg)](https://codecov.io/gh/ibis-project/ibis) |\n\n\nIbis is a toolbox to bridge the gap between local Python environments, remote\nstorage, execution systems like Hadoop components (HDFS, Impala, Hive, Spark)\nand SQL databases. Its goal is to simplify analytical workflows and make you\nmore productive.\n\nInstall Ibis from PyPI with:\n\n```sh\npip install ibis-framework\n```\n\nor from conda-forge with\n\n```sh\nconda install ibis-framework -c conda-forge\n```\n\nIbis currently provides tools for interacting with the following systems:\n\n- [Apache Impala](https://impala.apache.org/)\n- [Apache Kudu](https://kudu.apache.org/)\n- [Hadoop Distributed File System (HDFS)](https://hadoop.apache.org/)\n- [PostgreSQL](https://www.postgresql.org/)\n- [MySQL](https://www.mysql.com/)\n- [SQLite](https://www.sqlite.org/)\n- [Pandas](https://pandas.pydata.org/) [DataFrames](http://pandas.pydata.org/pandas-docs/stable/dsintro.html#dataframe)\n- [Clickhouse](https://clickhouse.yandex)\n- [BigQuery](https://cloud.google.com/bigquery)\n- [OmniSciDB](https://www.omnisci.com)\n- [PySpark](https://spark.apache.org)\n- [Dask](https://dask.org/) (Experimental)\n\nLearn more about using the library at http://ibis-project.org.\n",
```

Comment:
shouldn't this be in a separate README? (the long_description)

Reply:
It is, it's automatically generated.